static/js/controllers/medic-reporter-modal.js

```diff
@@ -49,8 +56,21 @@
         return url;
       };
 
-      var medicReporterBaseUrl = '/medic-reporter/_design/medic-reporter/_rewrite/';
-      checkAuth(medicReporterBaseUrl)
+      var medicReporterBaseUrl;
+      Settings()
+        .then(function(settings) {
+          if (!settings.muvuku_webapp_url) {
+            throw new Error({ status: 500, message: 'no medic-reporter url configured.' });
+          } else {
+            medicReporterBaseUrl = settings.muvuku_webapp_url.split('?').shift();
+            // Needs trailing slash, otherwise breaks!!
+            // https://github.com/medic/medic-reporter/issues/20
+            if (medicReporterBaseUrl.substr(-1) !== '/') {
+              medicReporterBaseUrl += '/';
+            }
+          }
+          return medicReporterBaseUrl;
+        }).then(checkAuth)
```

Comment:
So what's up with these changes? Is this fixing a bug that you found alone the way? Or some kind of tolerance for being deployed slightly differently due to e2e tests? The ticket this PR is against is just for e2e tests, so it's not clear why production code would be changing too.